source/solvers/heat_transfer_assemblers.cc

```diff
@@ -1172,3 +1172,92 @@ HeatTransferAssemblerFreeSurfaceRadiationVOF<dim>::assemble_rhs(
 
 template class HeatTransferAssemblerFreeSurfaceRadiationVOF<2>;
 template class HeatTransferAssemblerFreeSurfaceRadiationVOF<3>;
+
+template <int dim>
+void
+HeatTransferAssemblerVOFEvaporation<dim>::assemble_matrix(
+  HeatTransferScratchData<dim> &scratch_data,
+  StabilizedMethodsCopyData    &copy_data)
+{
+  // Loop and quadrature informations
+  const unsigned int n_q_points = scratch_data.n_q_points;
+  const unsigned int n_dofs     = scratch_data.n_dofs;
+
+  // Copy data elements
+  auto &local_matrix = copy_data.local_matrix;
+
+  // assembling local matrix and right hand side
+  for (unsigned int q = 0; q < n_q_points; ++q)
+    {
+      const double temperature = scratch_data.present_temperature_values[q];
+
+      std::map<field, double> field_value;
+      field_value[field::temperature] = temperature;
+
+      const double evaporative_heat_flux_jacobian =
+        this->evaporation_model->heat_flux_jacobian(field_value,
+                                                    field::temperature);
```

Comment:
It's ok to do it here I think, but it might be problematic if we start depending on multiple fields for the evaporative flux. Let's just keep that in mind